OpenNMT/onmt/Models.py

```diff
@@ -99,9 +98,9 @@ def __init__(self, opt, dicts):
 
 
     def forward(self, input, hidden, context, init_output):
-        emb = self.word_lut(input)
+        emb = self.word_lut(input).transpose(0, 1)
```

Comment:
I agree that an end-to-end change would be better. I was hoping for your feedback on this before messing with your setup too much, but now that it is merged, I'll make that a priority with the speed comparisons.